src/generics/constants/api-responses.js

```diff
@@ -56,4 +55,5 @@ module.exports = {
 	USER_ROLE_UPDATED: 'USER_ROLE_UPDATED',
 	USER_ROLE_DELETED: 'USER_ROLE_DELETED',
 	ENTITY_UPDATED: 'ENTITY_UPDATED',
+	ENTITY_TYPE_CREATION_FAILED: 'ENTITY TYPE CREATION FAILED',
```

Comment:
@MallanagoudaB Where are you using  this? have you made changes all the places we use this as well?